packages/foam-vscode/src/features/create-from-template.test.ts

```diff
@@ -149,6 +149,38 @@ describe('resolveFoamTemplateVariables', () => {
       await resolveFoamTemplateVariables(input, new Set(['FOAM_TITLE']))
     ).toEqual(expected);
   });
+
+  test('Correctly substitutes given variables that are missing in the template', async () => {

```

Comment:
```suggestion
  test('Adds FOAM_SELECTED_TEXT at end of template if it has variable has a value but is not referenced', async () => {
```

Also, we should have a test for when the `FOAM_SELECTED_TEXT` is used in the template but no text was selected (expected behavior: no content added. the variable should default to empty string).

Asking as by looking at the code I was unclear how that case would be handled

Reply:
> we should have a test for when the FOAM_SELECTED_TEXT is used in the template but no text was selected (expected behavior: no content added. the variable should default to empty string).

Was this test implemented? I can't find it